server/routes/label.js

```diff
@@ -0,0 +1,39 @@
+export default (app) => {
+  app
+    .get('/label', async (req, reply) => {
+      const labeles = await app.objection.models.label.query();
+      await reply.render('label/index', { currentUser: req.currentUser, labeles });
+    })
+    .get('/label/new', async (req, reply) => {
+      await reply.render('label/new', { data: {}, errors: [] });
+    })
+    .get('/label/edit/:id', async (req, reply) => {
+      const label = await app.objection.models.label.query().findById(req.params.id);
+      await reply.render('label/edit', { label, errors: [] });
+    })
+    .post('/label', async (req, reply) => {
+      try {
+        const newlabel = await app.objection.models.label.fromJson(req.body);
+        await app.objection.models.label.query().insert(newlabel);
+        await reply.redirect('/label');
+      } catch ({ data }) {
+        await reply.code(400).render('label/new', { data: req.body, errors: data });
+      }
+    })
+    .put('/label', async (req, reply) => {
+      try {
+        const { id, name } = req.body;
+        const updatedlabel = await app.objection.models.label.fromJson({ name });
```

Comment:
not sure if we need this, as we already declared `pickJsonSchemaProperties`

Reply:
do you mean just throw req.body in `fromJson` ? 
since we anyway destruct `id`, I thought why not destruct `name` as well

Reply:
but actually we dont need destruction then, thanks :)

Reply:
done